src/Test/Docs.hs

```diff
@@ -21,15 +21,14 @@ main = shaken noTest $ \args obj -> do
 
     config %> \_ -> do
         need ["shake.cabal"]
-        unit $ cmd "runhaskell Setup.hs configure" ["--builddir=" ++ obj "dist","--user"]
+        unit $ cmd "cabal configure" ["--builddir=" ++ obj "dist","--user"]
```

Comment:
OK, makes sense, I'm happy with that. I think there were some screwy cabal installs going around at one point that pushed me to runhaskell, but people can always upgrade cabal.